cmd/pump/pump.toml

```diff
@@ -31,6 +31,12 @@ pd-urls = "http://127.0.0.1:2379"
 # Set to `true` (default) for best reliability, which prevents data loss when there is a power failure.
 # sync-log = true
 #
+# the channel to buffer binlog meta, pump will block write binlog request if the channel is full
+# kv_chan_cap = 4194304
+#
+# if pump takes a long time to write binlog, pump will display the binlog meta information
+# slow_write_duration = 2
```

Comment:
better use a name **second, if you expect it as second

Reply:
I add a comment and don't change name, because I do't want to change the name if we change the definition of precision

Reply:
if you change the definition, then it's not compatible with the old version, you should not use the same name of config